sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs

```diff
@@ -44,6 +43,26 @@ public static HttpPipeline Build(ClientOptions options, params HttpPipelinePolic
             return new HttpPipeline(result.Transport, result.PerCallIndex, result.PerRetryIndex, result.Policies, result.Classifier);
         }
 
+        /// <summary>
+        /// </summary>
+        /// <param name="options"></param>
+        /// <param name="perCallPolicies"></param>
+        /// <param name="perRetryPolicies"></param>
+        /// <param name="messageClassifier">Classifier to be applied during classification for every operation on the client.</param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="messageClassifier"/> is null.</exception>
+        public static HttpPipeline Build1( // TODO: address ambiguous method resolution
```

Comment:
@christothes, @JoshLove-msft, any thoughts on how I could make this a proper overload?  I'm stumped on this one.  Thoughts I've considered: re-order parameters in the list, although the FDG recommends against that.  Adding more parameters, but they would be unnecessary.